py/desisim/io.py

```diff
@@ -939,8 +939,13 @@ def read_basis_templates(objtype, subtype='', outwave=None, nspec=None,
         log.info('Reading {} metadata.'.format(infile))
 
         if objtype.upper() == 'BAL': # non-standard data model
+            #meta = Table(fitsio.read(infile, ext=1, upper=True, columns=(
+            #    'SDSS_NAME', 'RA', 'DEC', 'PLATE', 'MJD', 'FIBERID')))
             meta = Table(fitsio.read(infile, ext=1, upper=True, columns=(
-                'SDSS_NAME', 'RA', 'DEC', 'PLATE', 'MJD', 'FIBERID')))
+                'SDSS_NAME', 'RA', 'DEC', 'PLATE', 'MJD', 'FIBERID','BI_CIV',
```

Comment:
I don't think we are using  'SDSS_NAME', 'RA', 'DEC', 'PLATE', 'MJD' columns at all, I left them here for now  just to ask if it is if I remove them from here, otherwise I remove them anyway in line L59 of bal.py.  @moustakas @paulmartini ?

Reply:
These columns were from the original BAL template set; please remove if they're obsolete.